resources/views/product/partials/options.blade.php

```diff
@@ -2,7 +2,7 @@
     <div class="flex flex-col space-y-3">
         @foreach ($product->options->sortBy('sort_order') as $option)
             <div>
-                @include('rapidez::product.partials.options.'.$option->type)
+                @includeIf('rapidez::product.partials.options.'.$option->type)
```

Comment:
Sure? All product option types are implemented?

Reply:
I guess you probably do want an error if you haven't implemented one?

Reply:
On a test environment; not on production. Maybe use this one: https://github.com/rapidez/blade-directives#includefirstsafe which does that.